src/interfaces/models.ts

```diff
@@ -96,8 +96,8 @@ export interface EntityI {
   intBlurbs?: string[]
   mveBlurbs?: string[]
   rehBlurbs?: string[]
-  amlBlurbs?: string[] | [string[], string[]?]
-  chgBlurbs?: string [] | [string[], string[]?]
+  amlBlurbs?: string[]
+  chgBlurbs?: string[]
```

Comment:
The new code isn't equivalent to the old. Is it OK?

Reply:
I updated the types for amlBlurbs and chgBlurbs but to me it still doesn't look right. Have a look.